documentation/asciidoc/topics/con_setting_up_monitoring.adoc

```diff
@@ -0,0 +1,20 @@
+[id='setting-up-monitoring_{context}']
+= Overview of Monitoring
+To monitor {brandname} clusters, you must enable `openshift-user-workload-monitoring` on {ocp}.
+
+The following high-level steps describe how to enable user workload monitoring:
+
+. Create a `ConfigMap` named `user-workload-monitoring-config` in the `openshift-user-workload-monitoring` project.
```

Comment:
Configurations applied to the user-workload-monitoring-config ConfigMap object are not activated unless a cluster administrator has enabled monitoring for user-defined projects. Enabling should be the fist step.

Reply:
Configuration of the user workload monitoring is optional step and we can provide juts link to the related documentation.